### ISSUE TITLE
FR: Half page scrolling support

### ISSUE THREAD

**Issue by beavailable** [readest/readest#62]
**Does your feature request involve difficulty completing a task? Please describe.**
When I read a book, my eyes are usually focusing on the middle of the page and I don't like reading the last few lines at the bottom, so I need scrolling down often (I can't just scroll to next page because I haven't finished reading the current page), it's a little inconvenient and somewhat annoying.

**Describe the solution you'd like**
If there are some keyboard shortcuts (such as `D` and `U`) for half page scrolling, it'd be much better for me.

**Comment by chrox** [comment]
> **Describe the solution you'd like** If there are some keyboard shortcuts (such as `D` and `U`) for half page scrolling, it'd be much better for me.

I like the Vimium keybinding.

**Comment by zangjiucheng** [comment]
To clarify the requirement, is it only triggered not in the scrolled mode and only with the single row mode?

Additionally, since `ctrl-d` and `ctrl-u` may conflict with some functional keys in the Arc browser, can I simplify it to just pressing `d` and `u`?

**Comment by chrox** [comment]
Yes it should be `d` and `u`. Ideally we should support as much Vimium keybindings as possible. 

![image](https://github.com/user-attachments/assets/0231b8df-c35e-4f8e-853a-9d2555a9e568)

**Comment by beavailable** [comment]
> To clarify the requirement, is it only triggered not in the scrolled mode and only with the single row mode?

Sorry I forgot to mention, I use scrolled mode all the time, so what I'd like is that it's triggered in the scrolled mode.
I don't care if it's triggered in other modes, but from what I can imagine, it doesn't affect any other functionalities.

> can I simplify it to just pressing `d` and `u`?

This is exactly what I expect.

**Comment by zangjiucheng** [comment]
I will try to update a PR today.
